pallets/elections/src/lib.rs

```diff
@@ -490,6 +516,8 @@ pub mod pallet {
                 }
             }
 
+            Self::unban();
```

Comment:
Technically this can be done in another place, to decrease size of the elections block. It needs to be done once per era.